### PR TITLE
move zabbix modules into their own namespace below monitoring/

### PR DESCRIPTION
**Pull request by eikef**
##### SUMMARY
This PR moves the zabbix_*-modules from modules/monitoring to modules/monitoring/zabbix and adds me as a maintainer to the two newly added zabbix_template and zabbix_proxy modules.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
zabbix_*

##### ANSIBLE VERSION
2.5.0

### CODE REVIEW

**Comment by jctanner** [comment]
bot_skip

**Comment by eikef** [comment]
@resmo, @D3DeFi, @cove, @harrisongu, @abulimov is this a good idea?

(there are two outstanding PRs which would be affected by this, both of which need a rebase anyway -- one a new module for maps in #23026 which is as yet unreviewed (working on it) which would just require a move of the new file, and one waiting on rebase and revison #23855 for another feature in zabbix_host which should be fairly easy to apply to a different path).

**Comment by resmo** [comment]
makes sense

shipit

**Comment by D3DeFi** [comment]
seems legit :)

**Comment by D3DeFi** [comment]
shipit

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -415,11 +415,13 @@ files:
   $modules/monitoring/stackdriver.py: bwhaley
   $modules/monitoring/statusio_maintenance.py: bhcopeland
   $modules/monitoring/uptimerobot.py: nate-kingsley
-  $modules/monitoring/zabbix_group.py: cove eikef harrisongu
-  $modules/monitoring/zabbix_host.py: cove eikef harrisongu D3DeFi
-  $modules/monitoring/zabbix_hostmacro.py: cave eikef
-  $modules/monitoring/zabbix_maintenance.py: abulimov eikef
-  $modules/monitoring/zabbix_screen.py: cove eikef harrisongu
+  $modules/monitoring/zabbix/zabbix_group.py: cove eikef harrisongu
```
Comment (pilou-) [inline .github/BOTMETA.yml]: Just for you information, the parent directory with your GitHub could be used:
```
$modules/monitoring/zabbix: eikef
$modules/monitoring/zabbix/zabbix_group.py: cove harrisongu
[...]
```

Comment (eikef) [reply .github/BOTMETA.yml]: Thank you for the pointer; pushed a commit to that effect :-)

**Review by D3DeFi — APPROVED** [review]

**Comment by D3DeFi** [comment]
shipit

**Pull request merged**